.github/workflows/manual_approve.yml

```diff
@@ -0,0 +1,18 @@
+name: Manual Approve
```

Comment:
Is this name right? 🤔 It's called Manual Approve but it looks like it's for triggering the docs command.

Reply:
Ah, yeah. That's because we can add more commands, so each command has it's own name:
https://github.com/cloudquery/cloudquery/pull/3027/files#diff-37ad898e2510969cb639d6ff3de59e1dd6c1c5fe2c8f6ce28661c47ce3eddd21R11

For example https://github.com/cloudquery/helm-charts/blob/07954f7161c507fe3988e9354db99274d1707a74/.github/workflows/manual_approve.yml has 2 different commands.

Reply:
Shouldn't it be called `Manual Commands` or something though?

Reply:
Yes, that's a much better name. Fixed in https://github.com/cloudquery/cloudquery/pull/3027/commits/d876a7b9771217e3a5f1b3794edb6a18d3c760b5